### DISCUSSION TITLE
Docs: NextImage document for png file

### DISCUSSION THREAD

**Discussion by adrian-cytrix** [vercel/next.js#69940] [Help]
### What is the documentation issue?

I tried to research about using png image as image source. but no luck on next js documentation. its external source images that can be config.

### Is there any context that might help us understand?

NextImage have issue on using png as source image.
error is not that clear how image can be trusted even its part of public/static file.

**Comment by samcx** [comment]
@adrian-cytrix What issue are you seeing when attempting to use a png file with `next/image`?

**Reply by adrian-cytrix** [reply]
![image](https://github.com/user-attachments/assets/d71cfa69-813c-44b5-ac7a-7d18deadef31)
I received this kind of error when using png as source in my nextjs project

**Reply by samcx** [reply]
@adrian-cytrix Can you create a :repro: so we can take a look?

**Comment by adrian-cytrix** [comment]
https://github.com/adrian-cytrix/school-dashboard

**Reply by samcx** [reply]
@adrian-cytrix Which Page does this error occurring on?

**Comment by adrian-cytrix** [comment]
any page under dashboard page group

**Reply by samcx** [reply]
@adrian-cytrix I was not been able to reproduce this. →

![CleanShot 2024-09-16 at 14 40 48@2x](https://github.com/user-attachments/assets/58dbe587-8a25-4316-a117-a431b41d296f)
